spire/humbug/actions.py

```diff
@@ -478,7 +474,7 @@ async def push_pack_to_journals_api(
             JournalEntryContent(
                 title=report.title,
                 content=report.content,
-                tags=report.tags,
+                tags=[tag for tag in report.tags if tag and len(tag) <= 1024],
```

Comment:
It not save, it checks only length of each tag, but I can still push 1,000,000 tags